adapters/http-vertx-base/src/test/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapterTest.java

```diff
@@ -410,6 +411,42 @@ public void testUploadCommandResponseFailsForDisabledTenant() {
         verify(metrics, never()).incrementCommandResponseDeliveredToApplication(eq("tenant"));
     }
 
+    /**
+     * Verifies that the adapter fails the upload of a command response with a 403 if the device is not registered and
```

Comment:
now the JavaDoc needs to be adapted to the new status code as well ...

Reply:
Done